interface/patient_file/report/custom_report.php

```diff
@@ -262,23 +264,13 @@ function zip_content($source, $destination, $content = '', $create = true)
                     $practice_logo = $plogo[$k];
                 }
 
-                echo "<div class='table-responsive'><table class='table' style='width:100%;'><tbody><tr><td align='left' class='align-middle'>";
-                ?>
-                <a href="javascript:window.close();"><h1><?php echo text($titleres['lname']) . ", " . text($titleres['fname']); ?></h1></a>
-                <?php
-                echo "<br clear='all' /></td><td align='right'>";
+                $logo = "";
                 if (file_exists($practice_logo)) {
                     $logo_path = $GLOBALS['OE_SITE_WEBROOT'] . "/images/" . basename($practice_logo);
-                    echo "<img class='h-auto' style='max-width:250px;' src='$logo_path'>"; // keep size within reason
-                    echo "</td><td>";
+                    $logo = "<img class='h-auto' style='max-height:8%;' src='". $logo_path . "'>";
```

Comment:
this might need an `attr()`
```
                    $logo = "<img class='h-auto' style='max-height:8%;' src='". attr($logo_path) . "'>";

Reply:
Works fine, adding just in case + testing, thanks